app/views/guests/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> Guest Info </h1>
+<p>Name: <%= @guest.name %></p>
+
+<h2> Room History </h2>
+<% @guest.rooms.each do |room| %>
+  <p><strong>Hotel: <%= room.hotel.name %></strong></p>
+  <p>Suite: <%= room.suite %></p>
+  <p>Rate: $<%= room.rate %></p>
+<% end %>
+
+<h4> Add a room: <h4>
+  <%= form_with url: "/guests/#{@guest.id}", method: :post, data: { turbo: false } do |f| %>
```

Comment:
Already mentioned this in the `routes.rb` file, but the path for this form should be `"/guests/#{@guest.id}/rooms"`